middleware/components/media-placeholder.js

```diff
@@ -171,10 +175,16 @@ class VideoPlaceholder extends Component {
 			];
 		}
 
+		if ( 'video' === type ) {
+			mediaEle = ( <video controls src={ this.state.mediaData.url }></video> );
```

Comment:
`mediaEle` should either be `mediaEl` or `mediaElement`

Reply:
Updated.